src/store/server/history/getters.ts

```diff
@@ -77,132 +77,6 @@ export const getters: GetterTree<ServerHistoryState, any> = {
             : 0
     },
 
-    getAllPrintStatusArrayAll(state, getters, rootState) {
-        const output: ServerHistoryStateAllPrintStatusEntry[] = []
-
-        state.jobs.forEach((current) => {
-            const index = output.findIndex((element) => element.name === current.status)
-            if (index !== -1) output[index].value += 1
-            else {
-                const displayName = i18n.te(`History.StatusValues.${current.status}`, 'en')
-                    ? i18n.t(`History.StatusValues.${current.status}`).toString()
-                    : current.status
-
-                const itemStyle = {
-                    opacity: 0.9,
-                    color: '#424242',
-                    borderColor: '#1E1E1E',
-                    borderWidth: 2,
-                    borderRadius: 3,
-                }
-
-                switch (current.status) {
-                    case 'completed':
-                        itemStyle['color'] = '#BDBDBD'
-                        break
-
-                    case 'in_progress':
-                        itemStyle['color'] = '#EEEEEE'
-                        break
-
-                    case 'cancelled':
-                        itemStyle['color'] = '#616161'
-                        break
-                }
-
-                output.push({
-                    name: current.status,
-                    displayName,
-                    value: 1,
-                    itemStyle,
-                    showInTable: !rootState.gui?.view.history.hidePrintStatus.includes(current.status),
-                })
-            }
-        })
-
-        return output
-    },
-
-    getAllPrintStatusArray(state, getters) {
```

Comment:
Removing this from the getter intruces a bug with the history list pannel.
![menu screenshot](image.png)
selecting the print status is not possible anymore. 
The getter is called in the `HistoryListPanel.vue` 
https://github.com/mainsail-crew/mainsail/blob/6caeca54f6371b3e978c73cb48a285123b8b5f0d/src/components/panels/HistoryListPanel.vue#L448

Reply:
i fixed it with the last commit